content/docs/Configuration/examples.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Examples 
+date: 2022-09-22
+weight: 1
+---
+
+# Configuration examples 
+
+## Examples for actions
+You can find detailed documentation for actions [here](/docs/actions).
+
+### get-current-version
+{{< details title="Getting version from specfile" >}}
+```yaml
+    get-current-version:
+    - grep -oP '^Version:\s+\K\S+' my-package.spec
+```
+{{< /details >}}
+
+{{< details title="Getting version for Python packages with python3" >}}
+```yaml
+    get-current-version:
+    - python3 setup.py --version
+```
+{{< /details >}}
+
+
+### post-upstream-clone
+{{< details title="Downloading specfile from the dist-git repo" >}}
+```yaml
+  post-upstream-clone:
+- "wget https://src.fedoraproject.org/rpms/my-package/raw/main/f/my-package.spec -O my-package.spec"
+```
+{{< /details >}}
+
+### create-archive
+{{< details title="Creating archive for Python packages with python3" >}}
+```yaml
+  create-archive:
+    - python3 setup.py sdist --dist-dir .
+    - bash -c "ls -1t ./my-package-*.tar.gz | head -n 1"
+```
+{{< /details >}}
+
+### fix-specfile
+{{< details title="Setting the Sources correctly in case of multiple Sources" >}}
+```yaml
+  fix-spec-file:
+  # define one of the Source variables correctly
+  - sed -i my_specfile_path -e "s/https.*only-vendor.tar.xz/my_correct_tarball_path/"
+  # fill in %release as if packit would have done it
+  - bash -c "sed -i my_specfile_path -r \"s/Release:(\s*)\S+/Release:\1${PACKIT_RPMSPEC_RELEASE}%{?dist}/\""
+```
+{{< /details >}}
+
+### changelog-entry
+{{< details title="Using changelog entry from a file" >}}
+```yaml
+  changelog-entry:
+  - cat .changelog_entry
+```
+{{< /details >}}
+
+
+## Examples for jobs
+You can find detailed documentation for jobs [here](/docs/configuration#supported-jobs).
+
+### copr_build
+
+{{< details title="Minimal configuration" >}}
+```yaml
+- job: copr_build
+  trigger: pull_request
+  targets:
+    - fedora-all
+```
+{{< /details >}}
+
+{{< details title="Using custom Copr project" >}}
+Configuring building in Copr project `@oamg/convert2rhel`:
+```yaml
+- job: copr_build
+  trigger: commit
+  branch: main
+  owner: "@oamg"
+  project: convert2rhel
+  targets:
+  - epel-6-x86_64
+  - epel-7-x86_64
+  - epel-8-x86_64
+```
+{{< /details >}}
+
+
+{{< details title="Using identifiers to differentiate the jobs" >}}
+```yaml
+jobs:
+- job: copr_build
+  trigger: pull_request
+  identifier: fedora
+  targets:
+    - fedora-all
+    
+- job: copr_build 
+  trigger: pull_request
+  specfile_path: epel8/python-specfile.spec
+  identifier: epel8
+  actions:
+    create-archive:
+      - python3 setup.py sdist --dist-dir ./epel8/
+      - bash -c "ls -1t ./epel8/*.tar.gz | head -n 1"
+  targets:
+    - epel-8
+```
+{{< /details >}}
+
+### tests
+
+{{< details title="Minimal configuration" >}}
```

Comment:
We can use the goal of the job for minimal configuration -- something like "I want to run FMF-based tests for my pull-request." / "I want to trigger the Koji build for new dist-git commits."/...